about.html

```diff
@@ -0,0 +1,153 @@
+        
+    <section id="about">
+      <div class="about-container1">
+      <h4 class="about-mp">Interior Design<br>
+         2022 Logo</h4>
+     <hr class="hr-title">
+        <ul class="">
+            <div class="about-box">
+            <p class="about-box-2">The logo of this symposium was designed by Phillip Hall in 2016.</p>
+            <div class="logo-img">
+            <h2>SUAVE INTERIORS.</h2>
+            
+        </div>
+        
+        </ul>
+     </div>
+    </section>
+    
+    <section class="see container-fluid text-center text-dark bg-white">
+        <div class="row pt-3 mb-5">
+          <div class="col-12">
+            <h4 class="text-center about-section-title">See the previous Symposium.</h4>
+            <hr class="hr-title">
+            <p class="about-info">Take a look at the last two datacamp webinars which took place<br>
+                 in Nairobi Kenya and Kampala Uganda.</p>
+          </div>
+        </div>
+    </section>
+        <div class="pst text-center" >
+            <div class="about-pst">
+                <div class="about-past">
+                <h3 class="number">2021</h3>
+                <p class="about-past-one">Interior design symposium 2021 kampala.</p>
+            </div>
+                <div class="about-pst-one">
+                    <div class="about-past">
+                    <h3 class="number">2020</h3>
+                        <p class="about-past-one">Interior design symposium 2021 Nairobi.</p>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+<!--     
+        <section class="partners-section">
+            <h4 class="partners-heading"><strong>Partners</strong></h4>
+            <hr />
+            <div class="partners-flex">
+              <div class="partners-list">
+                <img class="partner-image" src="https://raw.githubusercontent.com/tashisrael/Capstone1/main/images/partner1.png" alt="" />
+              </div>
+              <div class="partners-list">
+                <img class="partner-image" src="https://raw.githubusercontent.com/tashisrael/Capstone1/985f739e393fa40af073b99c2f8c89afa7a4e071/images/partner2.png" alt="" />
+              </div>
+              <div class="partners-list">
+                <img class="partner-image" src="https://raw.githubusercontent.com/tashisrael/Capstone1/985f739e393fa40af073b99c2f8c89afa7a4e071/images/partner3.png" alt="" />
+              </div>
+              <div class="partners-list">
+                <img class="partner-image" src="https://raw.githubusercontent.com/tashisrael/Capstone1/985f739e393fa40af073b99c2f8c89afa7a4e071/images/partner4.png" alt="" />
+              </div>
+              <div class="partners-list">
+                <img class="partner-image" src="https://raw.githubusercontent.com/tashisrael/Capstone1/985f739e393fa40af073b99c2f8c89afa7a4e071/images/partner5.png" alt="" />
+              </div>
+            </div>
+          </section> -->
```

Comment:
As suggested above, kindly avoid leaving pieces of commented code in your code base. The commit history can always take us back in time.